meetings/2023-11-15.md

```diff
@@ -0,0 +1,120 @@
+* eP: there was discussion about having a demo on this in the past. Would that be possible for the future? If there are multiple people who would be willing to demo this, it could be really valuable.
+* RG: will reach out to the main contributors to the spec
+
+
+### Special Topic Meetings
+URL: https://github.com/solid/specification/discussions/555
+
+* eP: [14 Nov discussion](https://github.com/solid/specification/issues/592) about Access Grants & ACLs
+* ...: issues have been raised about privacy of WebID. Would it be possible to use a pseudonym (non-correlatable) id?
+* ...: discussed ACP and Access Grants / Data Grants. Where there is an access grant, ACP delegates to the AG. With that, is the full ACP needed?
+* AC: Very briefly, full ACP is not needed if we end up delegating to Data Grants. The system doesn't need to be all that complicated.
+* eP: Currently in the solid spec, the clients MUST conform to WAC/ACP, which is a high bar. The regular applications only need to understand about data grants
+* ...: the access mgmt app is the only app that needs to understand the access policies/rules
+* ...: then only the authorization agent has to understand the complexity of access control
+* ...: perhaps we should reevaluate the role of WAC/ACP. Two main points:
+* ...: 1. can be beneficial if access mgmt is done in only one place
+* ...: 2. most apps can avoid interacting with access mgmt
+* ...: by making this simpler, we can make this more secure
+* AC: I agree, it is a high bar to assume that any app will be able to interact with access management system. Also anything what a regular app does can have a profound impacts. Anything what can keep regular apps away from interacting with access policies can be beneficial.
+* eP: What are the best next steps for this? Wait for working group? start discussing now?
+* PAC: I don't think we should wait for the WG. The main reason being that if WG is not chartered to work on that topic it will not pick it up. Given the number of depencencies, including the access control, especially the insufficient maturity level of those specifications. It is worthwhile to have this conversation before the working group starts. We should be clear what to include in the charter.
+* eP: I would propose that we coordinate a special topic meeting to discuss this further
+* ...: should we write something down first and then have the meeting?
+* AC: If you plan a meeting I will arrange to attend it. Outlining some of the specific details would be helpful to focus the conversation.
+
+ACTION: eP will expand on the [related issue](https://github.com/solid/specification/discussions/555) and will propose a date
+
+### W3C Solid WG Charter Proposal
+URL: https://github.com/solid/solid-wg-charter
+
+* PA: We received a [review](https://github.com/w3c/strategy/issues/377) from the TAG, which echo some remarks from the AC review.
+* ...: I don't know why we didn't get this review earlier, before sending it out to AC.
```

Comment:
@pchampin , back in March (2023-03-22) I've requested for a TAG review *prior to* AC review in https://github.com/solid/solid-wg-charter/issues/15 :

>As part of horizontal review, review from TAG should be requested to improve the charter prior to going to Members.

This topic was also on the CG agenda at least 4 times: https://github.com/search?q=repo%3Asolid%2Fspecification%20path%3A%2F%5Emeetings%5C%2F%2F%20https%3A%2F%2Fgithub.com%2Fsolid%2Fsolid-wg-charter%2Fissues%2F15&type=code and undoubtedly mentioned numerous times.

So, I know you/we've acknowledged and discussed this issue.

Why did we go to AC review without TAG review? Who made the call?

---

I would also add that every single one of the other points/considerations I've raised in that issue are literally the feedback we've received through AC reviews.

Please let me know if/how I can be of better service.

Reply:
> Why did we go to AC review without TAG review? Who made the call?

I request horizontal reviews on the charter back in June (see timeline of [this issue](https://github.com/w3c/strategy/issues/377#issue-1642474342)). Some groups responded, others (including the TAG) didn't). It is generally agreed that we can "timeout" on horizontal reviews, in order to keep things going.

In retrospect, I should have chased up the TAG in order to get their review... :disappointed: